### DISCUSSION TITLE
Community Support

### DISCUSSION THREAD

**Discussion by IAbstract** [AvaloniaUI/Avalonia#6518] [General]
I'm using the free version and depend on community support to determine if this is the solution for us. It seems to have great potential ... except for one thing: I can't seem to get any answers to my questions.

I'll never move beyond the free version if I cannot get answers.

**Comment by grokys** [comment]
Hi David. We're sorry you questions haven't been answered, but the core team are very busy and aren't always able to respond to every issue within a set timeframe: we're still a very small team. and the volume of issues/pull requests is very high.

You may have more luck on our gitter chat at https://gitter.im/AvaloniaUI/Avalonia which is frequented by lots of Avalonia enthusiasts who maybe don't read every issue opened on the repository.

**Comment by IAbstract** [comment]
Hey thanks for responding. Maybe it would be a good idea to have the web
page refer community intestinal to the gitter chat.
